impeller/entity/entity_pass.h

```diff
@@ -54,6 +54,10 @@ class EntityPass {
 
   void SetDelegate(std::unique_ptr<EntityPassDelegate> delgate);
 
+  void SetBoundsLimit(std::optional<Rect> bounds_limit);
```

Comment:
This should have a docstring.  In other comments you've added in this PR mention "absorbed clips".  We should mention that this what we are talking about.

Reply:
Done.